examples/advanced/custom_authentication/README.rst

```diff
@@ -0,0 +1,82 @@
+Example for Federated Policies
+==============================
+
+
+Overview
+--------
+
+The purpose of this example is to demonstrate following features of NVFlare,
+
+1. Run NVFlare in secure mode
+2. Demonstrate custom authentication policy
```

Comment:
Maybe we could explain what does custom authentication mean in NVFlare and how it is useful?
we could either add the explanation here or add that in documentation page and here we put a link reference to that.

Reply:
we can enhance these points in follow up PR.